src/DoctrineObject.php

```diff
@@ -443,16 +421,14 @@ protected function hydrateByReference(array $data, $object)
      * an identifier for the object. This is useful in a context of updating existing entities, without ugly
      * tricks like setting manually the existing id directly into the entity
      *
-     * @param  array  $data   The data that may contain identifiers keys
-     * @param  object $object
+     * @param  array $data The data that may contain identifiers keys
```

Comment:
I'm not familiar with Level 5. Is this PHPDoc still needed and doesn't it complain about the types of the elements?

Reply:
It's not needed for the type, but PHPCsFixer doesn't remove it because if the textual comment. So it's only kept for the description of the parameter.